man/systemd-vmspawn.xml

```diff
@@ -165,6 +165,18 @@
           <xi:include href="version-info.xml" xpointer="v255"/></listitem>
         </varlistentry>
 
+        <varlistentry>
+          <term><option>--qemu-net=</option><replaceable>user|tap|none</replaceable></term>
```

Comment:
i wonder if this wouldn't be better to default to off, i.e. `none`

Reply:
Now defaults to none in eae47ee9d9.